mods/common/scripts/utils.lua

```diff
@@ -12,3 +12,23 @@ IdleHunt = function(actor)
 		Trigger.OnIdle(actor, actor.Hunt)
 	end
 end
+
+---Adds a new mandatory objective that is localised and announces it via a message.
```

Comment:
Tbh "is localized" makes me think of map/mission design and world positioning. How about something like
```suggestion
---Adds a new mandatory objective and announces it via a translated message.
```

Reply:
I agree it's not the best, but it's the official term for the translation process.

Perhaps
```suggestion
---Adds a new mandatory objective, translates it and announces it via in-game chat message.
```

Reply:
could be changed by `localises it`